src/tests/EdFi.DataManagementService.Tests.E2E/Features/Resources/ReadResourcesValidation.feature

```diff
@@ -3,22 +3,19 @@ Feature: Resources "Read" Operation validations
         Background:
             Given the Data Management Service must receive a token issued by "http://localhost"
               And user is already authorized
-              And a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+              And a POST request is made to "/ed-fi/courseIdentificationSystemDescriptors" with
```

Comment:
This was always a mistake. Because the descriptor was not setup properly, the `POST` on line 19 was failing. But we didn't know it. With a change I made to the `id` extraction from the `location` header, the `School` created on line 18 ended up being in the `_id` field without being replaced by the `Course`'s `id` value. Thus the `GET` request was returning 200 instead of 404.